shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -43,23 +58,142 @@ std::vector<SkCanvas*> AndroidExternalViewEmbedder::GetCurrentCanvases() {
   return canvases;
 }
 
+SkRect AndroidExternalViewEmbedder::GetViewRect(int view_id) {
+  EmbeddedViewParams* params = &view_params_[view_id];
+  FML_CHECK(params);
+  return SkRect::MakeXYWH(params->offsetPixels.x(),                          //
```

Comment:
Nit: I am unsure about this. Is the offset already in physical device coordinates? You can refer to the stuff in embedder_external_view_embedder.

Reply:
It is. I was also surprised. This is also true for iOS. I added a TODO to investigate further. cc @cyanglaz

Reply:
It turns out this is incorrect. I filed https://github.com/flutter/flutter/issues/59821